chalicelib/checks/wfr_checks.py

```diff
@@ -475,19 +757,23 @@ def spot_failed_metawfrs(connection, **kwargs):
         check.full_output = {}
         return check
 
+    ### Check
+    # query
     query = '/search/?type=MetaWorkflowRun' + \
             ''.join(['&final_status=' + st for st in ['failed']])
     query += ''.join(['&meta_workflow.title=' + mwf for mwf in default_pipelines_to_run])
     search_res = ff_utils.search_metadata(query, key=my_auth)
```

Comment:
This code pattern is repeated a lot - another good candidate for a helper function.

Reply:
that is definitely a good idea